build_and_upload.sh

```diff
@@ -43,37 +68,69 @@ array=(osx-64 linux-64 win-64 linux-32 win-32)
 for i in "${array[@]}"
 do
     echo Uploading: $i;
-	binstar upload -u bokeh $i/bokeh*$date*.tar.bz2 -c dev --force;
+    binstar upload -u bokeh $i/bokeh*$date*.tar.bz2 -c dev --force;
 done
 
-if [ "$tag_flag" = "1" ]; then
-    version=`git describe --tags`
-else
-    version=`python scripts/get_bump_version.py`
-fi
-
 #create and upload pypi pkgs to binstar
-
 #zip is currently not working
 
 BOKEH_DEV_VERSION=$version.dev.$date python setup.py sdist --formats=gztar
 binstar upload -u bokeh dist/bokeh*$date* --package-type pypi -c dev --force;
 
-echo "I'm done uploading"
+echo "I'm done uploading to binstar"
 
-#clean up
-for i in "${array[@]}"
-do
-    rm -rf $i
-done
-
-rm -rf dist/
-rm using_tags.txt
+########################
+#   General clean up   #
+########################
 
-#####################
-#Removing on binstar#
-#####################
+#clean up platform folders
+if [ $clean == true ]; then
+    for i in "${array[@]}"
+    do
+        rm -rf $i
+    done
+    rm -rf dist/
+else
+    echo "Not cleaning the packages."
+fi
 
+#and the additional building stuff
+rm -rf build/
+rm -rf bokeh.egg-info/
+rm -rf record.txt
+rm -rf versioneer.pyc
+rm -rf __conda_version__.txt
+rm -rf bokeh/__conda_version__.py
+rm -rf bokeh/__conda_version__.pyc
+rm -rf bokeh/__pycache__/__conda_version__.pyc
```

Comment:
Yes... I agree... well, in fact, it would not matter when things get built by TravisCI or Binstar... because it is a disposable env...